.github/workflows/test.yml

```diff
@@ -50,6 +50,9 @@ jobs:
   eslint:
     name: ESLint (Node v${{ matrix.node }})
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [16]
```

Comment:
You need to use this in the setup-node section if you want to pin it. I removed that line when I unpinned from 12. By default it should use the current LTS, which is 16, already

Reply:
@nschonni, Do you know how to interpolate the version that gets used into the output? Without the matrix config, the line `ESLint (Node v${{ matrix.node }})` returns null for the `${{ matrix.node }}` value. Maybe the `runner` has a param for the version of Node that gets loaded?

Reply:
Yeah, I should have cleaned up the title when I removed the matrix value in the setup-node `with` section.
I would probably suggest cleaning up the title unless you want to keep it pinned.

Reply:
@nschonni I am OK with keeping it pinned. It'll be valid for another year plus :)

Reply:
OK, you'd just need to add back the line like on https://github.com/A11yance/aria-query/pull/312/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R23 to get it to pick it up

Reply:
Good call. Fixed in #314